nix/requirements_override.nix

```diff
@@ -116,6 +116,12 @@ in skipOverrides {
     buildInputs = old.buildInputs ++ [ self."Cython" ];
   };
 
+  "bugbug" = self: old: {
+    patchPhase = ''
+      sed -i 's/python-dateutil==2.8.0/python-dateutil/' requirements.txt
```

Comment:
You're patching your own deps ? :smile:

Reply:
Unfortunately there is something else forcing python-dateutil to 2.6.1, so I can't use the required version from bugbug.